card/lib/card/format.rb

```diff
@@ -510,26 +522,26 @@ def default_item_view
       :name
     end
 
-
     #
     # ------------ LINKS ---------------
     #
 
     def add_class options, klass
-      options[:class] = [ options[:class], klass ].flatten.compact * ' '
+      options[:class] = [options[:class], klass].flatten.compact * ' '
     end
 
-
     def unique_id
       "#{card.key}-#{Time.now.to_i}-#{rand(3)}"
     end
 
-    def format_date date, include_time = true
-      # Must use DateTime because Time doesn't support %e on at least some platforms
+    def format_date date, include_time=true
+      # using DateTime because Time doesn't support %e on some platforms
       if include_time
-        DateTime.new(date.year, date.mon, date.day, date.hour, date.min, date.sec).strftime("%B %e, %Y %H:%M:%S")
+        DateTime.new(
```

Comment:
Do not use `DateTime.new.strftime` without zone. Use one of `Time.zone.new.strftime`, `DateTime.new.strftime.current`, `DateTime.new.strftime.in_time_zone`, `DateTime.new.strftime.utc`, `DateTime.new.strftime.getlocal`, `DateTime.new.strftime.iso8601`, `DateTime.new.strftime.jisx0301`, `DateTime.new.strftime.rfc3339`, `DateTime.new.strftime.to_i`, `DateTime.new.strftime.to_f` instead.